content/blog/2020/01/2020-01-23-ui-plugins.adoc

```diff
@@ -0,0 +1,701 @@
+like HTML document traversal and manipulation, event handling, animation, and Ajax much simpler with an easy-to-use API
+that works across a multitude of browsers. With a combination of versatility and extensibility, jQuery has changed the
+way that millions of people write JavaScript.
+
+- https://github.com/jenkinsci/bootstrap4-api-plugin[bootstrap4-api-plugin]:
+Provides https://getbootstrap.com/[Bootstrap 4] for Jenkins Plugins. Bootstrap is -- according to their self-perception --
+the world’s most popular front-end component library to build responsive, mobile-first projects on the web. It is
+an open source toolkit for developing with HTML, CSS, and JS. Developers can quickly prototype their ideas or
+build entire apps with their Sass variables and mixins, responsive grid system, extensive prebuilt components, and powerful plugins
+built on jQuery.
+
+- https://github.com/jenkinsci/data-tables-api-plugin[data-tables-api-plugin]:
+Provides https://datatables.net[DataTables] for Jenkins Plugins.
+DataTables is a plug-in for the jQuery Javascript library. It is a highly flexible tool, built upon the foundations
+of progressive enhancement, that adds all of these advanced features to any HTML table:
+** Previous, next and page navigation
+** Filter results by text search
+** Sort data by multiple columns at once
+** DOM, Javascript, Ajax and server-side processing
+** Easily theme-able
+** Mobile friendly
+
+- https://github.com/jenkinsci/echarts-api-plugin[echarts-api-plugin]:
+Provides https://echarts.apache.org/en/index.html[ECharts] for Jenkins Plugins. ECharts is an open-sourced
+JavaScript visualization tool to create intuitive, interactive, and highly-customizable charts. It
+can run fluently on PC and mobile devices and it is compatible with most modern
+Web Browsers.
+
+- https://github.com/jenkinsci/font-awesome-api-plugin[font-awesome-api-plugin]:
+Provides https://fontawesome.com[Font Awesome] for Jenkins Plugins. Font Awesome has vector icons and social logos,
+according to their self-perception it is the web's most popular icon set and toolkit. Currently, it contains more than
+1,500 free icons.
+
+- https://github.com/jenkinsci/popper-api-plugin[popper-api-plugin]
+Provides https://popper.js.org[Popper.js] for Jenkins Plugins. Popper can
+easily position tooltips, popovers or anything else with just a line of code.
+
+- https://github.com/jenkinsci/plugin-util-api-plugin[plugin-util-api-plugin]: This small plugin provides
+some helper and base classes to simplify the creation of reporters in Jenkins. This plugin also
+provides a set of architecture rules that can be included in an architecture test suite of your plugin.
+
+== Required changes for a plugin POM
+
+In order to use these plugins you need to add them as dependencies in your plugin pom. You can use the following snippet
+to add them all:
+
+// FIXME: replace the beta tag once the 1.0 releases are done
```

Comment:
Once you've released 1.0, I would recommend adding these to the [bom](https://github.com/jenkinsci/bom)

Reply:
Yes, I can do. I thought the bom should be used only for plugins that are widely used though...